pkg/daemon/update.go

```diff
@@ -419,6 +419,12 @@ func calculatePostConfigChangeActionFromFileDiffs(diffFileSet []string) (actions
 		imageRegistryAuthFile,
 		"/var/lib/kubelet/config.json",
 	}
+	directoriesPostConfigChangeActionNone := []string{
+		// We probably don't want to use this exact path because NMState is
+		// planning to add a service that applies configs from it too, and we
+		// want to make sure our service is the only one processing the configs.
+		"/etc/nmstate/openshift",
```

Comment:
Why chose this dir vs, say, a directory out of a well known path? Potential for conflicts are low but I guess it would be less if we chose a different path.

Reply:
Do you have a suggestion on another location? I don't particularly care where the files are, but since this is OpenShift-specific NMState configuration I thought an openshift directory under /etc/nmstate made sense.

Reply:
I will note that we had a discussion about naming this something like "/etc/nmstate/techpreview-openshift" to make sure it's clear to anyone using this functionality that it is currently tech preview. We'll be documenting that too, but not everyone reads the docs, unfortunately.

Reply:
We already use paths like `/etc/cno` and `/etc/ovnk`. `/etc/mco` also exists and I guess a potentially new `/etc/openshift` comes to mind as well. If none of these seem plausible to you, then well I guess I don't have really any suggestion so we can go with what you have here.